src/material/snack-bar/snack-bar-container.ts

```diff
@@ -247,8 +247,14 @@ export class MatSnackBarContainer extends BasePortalOutlet
     if (!this._announceTimeoutId) {
       this._ngZone.runOutsideAngular(() => {
         this._announceTimeoutId = setTimeout(() => {
-          const inertElement = this._elementRef.nativeElement.querySelector('[aria-hidden]');
-          const liveElement = this._elementRef.nativeElement.querySelector('[aria-live]');
+          const { nativeElement } = this._elementRef;
+          const inertElement = nativeElement.querySelector('.mat-snack-bar-aria-hidden');
+          const liveElement = nativeElement.querySelector('.mat-snack-bar-aria-live');
+
+          if (nativeElement.querySelectorAll('[aria-live]').length > 1) {
```

Comment:
Is this something we have to worry about?

Reply:
Technically no. We do tell users that we are using an aria-live region [in our docs](https://material.angular.io/components/snack-bar/overview). I can remove it if we don't want to do this

Reply:
I think we're fine with not checking for this. Even if somebody is doing it, chances are that they know what they're doing.

Reply:
sgtm 👍🏽